### PR TITLE
Update various dependencies

### PR DESCRIPTION
**Pull request by olemarkus**

### CODE REVIEW

File: etcd-manager/go.mod [hunk 1]
```diff
@@ -2,31 +2,33 @@ module sigs.k8s.io/etcdadm/etcd-manager
...
 	google.golang.org/grpc v1.47.0
 	gopkg.in/gcfg.v1 v1.2.3
-	k8s.io/apimachinery v0.24.2
-	k8s.io/client-go v0.24.2
-	k8s.io/klog/v2 v2.60.1
-	k8s.io/kops v1.24.0-beta.2
-	k8s.io/mount-utils v0.24.2
-	k8s.io/utils v0.0.0-20220210201930-3a6ce19ff2f9
+	k8s.io/apimachinery v0.24.3
+	k8s.io/client-go v1.5.2
```
Comment (hakman) [inline etcd-manager/go.mod:27]: Any idea why this changed to v1.5.2 and why replace was needed?
```suggestion
	k8s.io/client-go v1.5.2
```

Comment (olemarkus) [reply etcd-manager/go.mod:27]: I don't really understand any of these client-go changes that are required. It seems to come from replace being removed in kOps. And why 1.5.2? no idea. kubetest2-kops pulls in version v11. Nothing I do makes it stick with the proper version.

Comment (hakman) [reply etcd-manager/go.mod:27]: Solution seems to be to switch to `k8s.io/kops v1.25.0-alpha.1`, or do another 1.24 release.

Comment (olemarkus) [reply etcd-manager/go.mod:27]: Will wait for our next release then.

**Review by hakman — APPROVED** [review]

**Pull request merged**